content/en/continuous_integration/pipelines/gitlab.md

```diff
@@ -58,11 +58,81 @@ Configure the integration on a [project][101] or [group][102] by going to **Sett
 [101]: https://docs.gitlab.com/ee/user/admin_area/settings/project_integration_management.html#view-projects-that-use-custom-settings
 [102]: https://docs.gitlab.com/ee/user/project/integrations/index.html#manage-group-default-settings-for-a-project-integration
 
+Fill in the integration configuration settings:
+
+**Active**
+: Enables the integration.
+
+**Datadog site**
+: Specifies which [Datadog site][1] to send data to.<br/>
+**Default**: `datadoghq.com`<br/>
+**Selected site**: {{< region-param key="dd_site" code="true" >}}<br/>
+
+**API URL** (optional)
+: Allows overriding the API URL used for sending data directly, only used in advanced scenarios.<br/>
+**Default**: (empty, no override)
+
+**API key**
+: Specifies which [Datadog API key][2] to use when sending data.
+
+**Enable CI Visibility**
+: Enables pipeline tracing. This box should be checked.
```

Comment:
This is the new field. I didn't find a better way than duplicate the entire section for all tabs, I tried adding a shortcode but that's doesn't seem possible because of the `dd_site` param that we need to render, let me know if there is a better way !

Reply:
this seems fine